Libplanet/Blocks/BlockHeader.cs

```diff
@@ -150,6 +150,8 @@ BlockHash Hash
         /// <inheritdoc cref="IBlockMetadata.TxHash"/>
         public HashDigest<SHA256>? TxHash => _preEvaluationBlockHeader.TxHash;
 
+        public BlockCommit? LastCommit => _preEvaluationBlockHeader.LastCommit;
```

Comment:
Is `LastCommit` required in both `Block` and `BlockHeader`?

Reply:
Recording to #1492, `BlockHeader` and `Block<T>` are implementations of `IBlockHeader` and `IBlock<T>`. 
`LastCommit` is a property of `IBlockMetadata` so it is required separately.

Reply:
But we have `BlockHeader` property in `Block<T>`. You should use like
```csharp
    public BlockCommit? LastCommit => BlockHeader.LastCommit;
```
see also https://github.com/planetarium/libplanet/blob/9e874b24a0c2b0cf188adbf9bf52f20a1cfdb613/Libplanet/Blocks/Block.cs

Reply:
Aha, you mean https://github.com/planetarium/libplanet/pull/1969/files/4c8ed9c8645051edf523af07a5cb7d73a84825cc#diff-81f06fcc7609e3eaf7fa94e98ce460fdaf9ec8923766e1dee100c7e03af88aebR146 here? I'm confused you made a claim on the `BlockHeader` lol. Yes, you're right. I mistake it. I'll amend it.

Reply:
Oh I should've leave this comment in `Block.cs` :)

Reply:
c6f5c3f2e070fa222a0ea450e13590a695973840 amended